### ISSUE TITLE
imperfect rendering

### ISSUE THREAD

**Issue by zadam** [michael-brade/LaTeX.js#16]
Hello, I wasy trying out your project and it kinda works, but rendering is slightly off compared to the playground:

![image](https://user-images.githubusercontent.com/617641/44989734-1c9c7580-af8f-11e8-9398-78c9aef0cfa3.png)

I used example script from the README:

```javascript
var text = document.getElementById("content").value;
var generator = new latexjs.HtmlGenerator({ hyphenate: false, bare: true })

var dom = latexjs.parse(text, { generator: generator }).dom()

document.body.appendChild(dom)
```

By inspecting playground it looks like there's some different CSS which need to be loaded into the page but that's not mentioned in the README.

Otherwise this is really cool project!

**Comment by michael-brade** [comment]
Yeah, indeed, you guys are really beating me to the docs! :) In this case, "`bare: true`" is the reason for the difference, you won't get any headers that way, only the contents of the `<body>`. If you set `bare: false`, you will have to display the result in a frame or take the DOM apart and insert stuff manually (but that could lead to conflicts and differences, too). I will write about that soon.

**Comment by michael-brade** [comment]
Ok, I figuerd I'm going to do it properly and turn it into a webcomponent. Give me some more time...

**Comment by michael-brade** [comment]
Thank you so much for this report! It made me create the web component, and I believe this is a huge improvement :smile: I now released 0.11.1. Hope you like it, too.
